firebase-firestore/src/main/java/com/google/firebase/firestore/model/value/ServerTimestampValue.java

```diff
@@ -37,7 +37,7 @@
  * <li>They sort after all TimestampValues. With respect to other ServerTimestampValues, they sort
  *     by their localWriteTime.
  */
-public final class ServerTimestampValue extends FieldValue {
+public final class ServerTimestampValue {
```

Comment:
I'm surprised this works. Is the only usage of this in `UserDataWriter`?

Reply:
The class is only used in `UserDataWriter` and `ProtoValues`. `isServerTimestamp()` is used in a couple other places.